packages/curve-ui-kit/src/themes/mui-tooltip.ts

```diff
@@ -0,0 +1,24 @@
+import type { Components } from '@mui/material/styles'
+import type { TypographyOptions } from '@mui/material/styles/createTypography'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import { handleBreakpoints } from './basic-theme'
+import { getShadow } from './basic-theme/shadows'
+import type { DesignSystem } from './design'
+
+const { Spacing } = SizesAndSpaces
+
+export const defineMuiTooltip = (design: DesignSystem, typography: TypographyOptions): Components['MuiTooltip'] => ({
+  styleOverrides: {
+    tooltip: {
+      backgroundColor: design.Layer[3].Fill,
+      boxShadow: getShadow(design, 2),
+      minWidth: '27.5rem', // hardcoded 440px in the design
```

Comment:
`SizesAndSpaces.MinWidth.tooltip` in `1_sizes_spaces`?

Reply:
Good point!